FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.m

```diff
@@ -147,6 +160,38 @@ - (FIRTrace *)activeTrace {
   return self.backgroundSessionTrace;
 }
 
+- (void)startTrackingNetwork {
+  self.networkType = firebase_perf_v1_NetworkConnectionInfo_NetworkType_NONE;
+
+#if TARGET_OS_IOS
+  if (@available(iOS 12, *)) {
+#elif TARGET_OS_TVOS
+  if (@available(tvOS 12, *)) {
+#endif
+    dispatch_queue_attr_t attrs = dispatch_queue_attr_make_with_qos_class(
+        DISPATCH_QUEUE_SERIAL, QOS_CLASS_UTILITY, DISPATCH_QUEUE_PRIORITY_DEFAULT);
+    self.monitorQueue = dispatch_queue_create("com.google.perf.network.monitor", attrs);
```

Comment:
Nit: `com.google.firebase.perf.network.monitor`, I see that's the most patterns in sdk :)

Reply:
Done.